File: DS Assignments/ProgrammingAssignment4/ProgrammingAssignment4/ProgrammingAssignment4/Controller.cpp
#include "Controller.h"


Controller::Controller()
{
}
void Controller::UCLoadData()
{
	emp.loadData();
}
	
void Controller::UCDisplayDepth()
{
	emp.displayDepth();
}
void Controller::UCDisplayBreath()
{
	emp.displayBreath();
}
void Controller::UCDisplayLevel()
{
	emp.displayLevel();
}
void Controller::UCSearchRole()
{
	emp.searchRole();
}
void Controller::UCSearchName()
{
	emp.searchName();
}
void Controller::UCAdd()
{
	emp.addEmployee();
}
void Controller::UCRemove()
{
	emp.removeEmployee();
}
void Controller::UCSave()
{
	emp.save();
}

File: DS Assignments/ProgrammingAssignment1/ProgrammingAssignment1/src/programmingassignment1/Controller.java
// COSC 2320 Summer 2015
// Ali Elsaadi
// Progamming Assignment 1
// This is my own work; I will not post
package programmingassignment1;

import java.io.FileNotFoundException;


public class Controller {
    Employee emp = new Employee();
    
    public void UCLoadData()
    {
        emp.loadData();
        System.out.println("Data loaded");
        
    }
    public void UCDisplay()
    {
        emp.displayHT();
    }
    public void UCADD()
    {
        emp.addEmployee();
    }
    public void UCRemove()
    {
        emp.removeEmployees();
    }
    public void UCSave() throws FileNotFoundException
    {
        emp.save();
    }
}
